### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Amazing3D**
Added some of the details from the ginge braindump https://gist.github.com/ginge/bdaa7c20f23871980b9e10db91945825

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,82 @@
...
+|Buttons:         |Y                |Y                       |Y            |N              |
+|Gyro:            |N                |N                       |N            |N              |
+|Magnetometer:    |N                |N                       |N            |N              |
+|Vibrate:         |N                |Y                       |Y            |N              |
+|Backlight:       |N                |Y                       |Y            |N              |
+|Power Management:|N                |N                       |N            |N              |
+|Bluetooth:       |N                |Y                       |N            |N              |
+|Overall (%):     |1%               |20%                     |10%          |0%             |
+
+## Roadmap
```
Comment (ginge) [inline README.md]: I think this section is too aspirational at the moment. Not much of the below is correct any more (and the prefix [PR] in he gist was the state of that particular task. I'm not sure tracking that sort of stuff in the readme is ideal.
Remove whole section please

File: README.md [hunk 2]
```diff
@@ -75,6 +141,17 @@ system applications.  We break down these components as follows:
...
+### Code style guide
+
+logging:
+- DRV_LOG for hw
+- KERN_LOG for rcore drivers
+- SYS_LOG for rcore processes
+- APP_LOG is automatic for apps(edited)
+
+No style wars.
+Do what's best.
```
Comment (ginge) [inline README.md]: Appears on one line on the final output

File: README.md [hunk 3]
```diff
@@ -1,16 +1,82 @@
...
+|:---:            |:---:            |:---:                   |:---:        |:---:          |
+|Display:         |N                |Y                       |N            |N              |
+|Flash:           |Y                |Y                       |Y            |N              |
+|Buttons:         |Y                |Y                       |Y            |N              |
+|Gyro:            |N                |N                       |N            |N              |
+|Magnetometer:    |N                |N                       |N            |N              |
+|Vibrate:         |N                |Y                       |Y            |N              |
+|Backlight:       |N                |Y                       |Y            |N              |
+|Power Management:|N                |N                       |N            |N              |
+|Bluetooth:       |N                |Y                       |N            |N              |
```
Comment (ginge) [inline README.md]: For Snowy BT I would put Y (Alpha)

File: README.md [hunk 4]
```diff
@@ -1,16 +1,82 @@
...
+- music protocol api
+- phone notification layout
+- notification gui
+##### [DONE]
+- app loading/ watchface loading differentiation
+- background workers
+- PDC drawing doesn't layer offset
+- lots of unalloc functions (see matrix)
+
+## FAQ
```
Comment (ginge) [inline README.md]: This is great :)

File: README.md [hunk 5]
```diff
@@ -75,6 +141,17 @@ system applications.  We break down these components as follows:
 
 * **System applications.** We'll, uh, get there when we have some.  Yeah.
 
+### Code style guide
+
+logging:
+- DRV_LOG for hw
+- KERN_LOG for rcore drivers
+- SYS_LOG for rcore processes
```
Comment (ginge) [inline README.md]: SYS_LOG for rcore/rwatch processes

**Comment by jwise** [comment]
How do we feel about starting to split things out into more documentation files? Perhaps doc/FAQ.md; doc/status.md; and then docs/*_build.md move to doc/build/*.md (note that directory names are usually singular)?

**Comment by ginge** [comment]
i'm usually a fan of one page summaries without turning the whole thing into a wiki.
Is there compromise somewhere? Move the matrix, keep the mini FAQ. Link to a bigger FAQ with more comprehensive answers? (not that I keep pasting the same thing over in the discord ;) )

**Pull request merged**